src/lucky/params.cr

```diff
@@ -492,4 +523,20 @@ class Lucky::Params
   private def query_params
     request.query_params
   end
+
+  private def get_all_json(key : String)
+    val = parsed_json[key]?
+    return if val.nil?
+
+    val.as_a?.try(&.map(&.to_s)) || [val.to_s]
+  end
+
+  private def get_all_params(params, key : String)
+    vals = params.fetch_all(key + "[]")
+    if vals.any?
+      vals
+    else
+      nil
+    end
```

Comment:
What do you think about this?

```suggestion
    params.fetch_all(key + "[]").try &.any?
```

Reply:
Doesn't `#any?` return a boolean?  `#fetch_all` always returns an array, the goal is to return nil if the array is empty